python/ray/tests/test_multiprocessing.py

```diff
@@ -484,3 +485,8 @@ def f(args):
 
     pool = Pool(5, maxtasksperchild=1)
     assert len(set(pool.map(f, range(20)))) == 20
+
+
+if __name__ == "__main__":
+    import pytest
+    sys.exit(pytest.main(["-v", __file__]))
```

Comment:
This test wasn't being run by bazel before.

Reply:
You need to add a main to the file for bazel to run. Bazel just runs the file, it doesn't know anything about pytest.

Reply:
I commented out the broken tests for now. They seem asyncio related.